### DISCUSSION TITLE
new input manager를 반드시 사용해야 하나요?

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#541] [5. Q&A]
new input manager를 반드시 사용해야 하나요? 

![image](https://user-images.githubusercontent.com/50188795/160519986-8094ce53-bd0f-4dca-898c-7bb517378d9a.png)

**Comment by Zepeto-TechEvangelist** [comment]
기존 Input Manager도 사용하고 싶다면 Both를 선택해서 사용 가능합니다.

![스크린샷 2022-03-29 오후 6 19 30](https://user-images.githubusercontent.com/100118650/160581263-af12a5cc-a0f2-4795-8037-81d22bedef20.png)

**Reply by smilejsu82** [reply]
답변 감사합니다 ~
